lessons/AI/ai-statistical-lang-models/langs/en-us/pages/tic-tac-toe.adoc

```diff
@@ -0,0 +1,51 @@
+= Soekia & Tic-Tac-Toe
+
+Respond to the prompts below by interacting with @link{http://Soekia.ch/GPT/?lang=en}
+
+== Documents and Text Generation
+
+@n Describe what you see in the Documents panel. @fitb{}{@ifsoln{There are four documents (A, B, C, and D). Each one is a different color.}}
+
+@fitb{}{@ifsoln{Each document narrates a different game of tic-tac-toe using a shorthand notation.}}
+
+@n Scroll left to the blue "Generate text" panel. Click the right-facing arrow (&#x27A4;) to play tic-tac-toe.
+
+@n Some of the moves in your tic tac toe game may be highlighted with different colors (green, pink, orange, etc). If this is not the case, click the eye icon beneath the generated text. Hover your mouse over one or two of the highlighted moves in the tic-tac-toe game. What appears? @fitb{}{}
+
+@fitb{}{}
+
+@fitb{}{}
+
+@n Compare your tic-tac-toe game with a classmate's tic-tac-toe game. How alike or different are the two games? Why do you think this is the case? @fitb{}{}
+
+@fitb{}{}
```

Comment:
Please provide guidance for teachers here. I am not able to guess what will happen if two different students play the game.